lib/Core/arrayContains.ts

```diff
@@ -0,0 +1,8 @@
+export default function arrayContains<T>(array: readonly T[], value: T) {
+  for (let i = 0; i < array.length; ++i) {
+    if (array[i] === value) {
+      return true;
+    }
+  }
+  return false;
+}
```

Comment:
Instead remove and replace with `Array.prototype.includes` (only used in 1 place in terriajs)